Cargo.toml

```diff
@@ -19,6 +19,7 @@ http = ["surrealdb/http"]
 http-compression = []
 # Private features
 has-storage = []
+ml = ["surrealml-core", "surrealdb/ml"]
```

Comment:
Is this a private feature? It's in the default features, which suggests otherwise. If it's public, we should move it up, into the public features section.